packages/server/modules/gatekeeper/services/checkout.ts

```diff
@@ -88,15 +88,19 @@ export const completeCheckoutSessionFactory =
     await upsertWorkspaceSubscription({
       workspaceSubscription
     })
+
+    const previousPlan = previousWorkspacePlan
```

Comment:
Kinda weird that `previousPlan` is not actually the previous plan, but a wrapper object w/ another key called `previousPlan` that actually holds it. Why the unnecessary extra level of nesting? The variable seems redundant too, why not just use `previousWorkspacePlan`

```
await emitEvent({
  payload: {
    workspacePlan: { .. },
    previousPlan: previousWorkspacePlan
  }
})
```